### ISSUE TITLE
only one forward test

### ISSUE THREAD

**Issue by hljjjmssyh** [NVIDIA/TensorRT-LLM#158]
Can you provide examples of model forward propagation? 
Most examples involve directly calling the generate function to obtain results. 
I would like to test the model by obtaining the logits from one forward pass. How can I do this?

**Comment by byshiue** [comment]
I am not sure what's kind of example you want to see. The workflow you mention is put in https://github.com/NVIDIA/TensorRT-LLM/blob/release/0.5.0/tensorrt_llm/runtime/generation.py. It may be helpful.

**Comment by byshiue** [comment]
Close this bug because it is in-activate. Feel free to reopen it if needed.
